### DISCUSSION TITLE
Gray theme

### DISCUSSION THREAD

**Discussion by Jopp-gh** [alainm23/planify#1625] [Ideas]
**Is your feature request related to a problem? Please describe.**
I don't get why Planify has a dark+blue accent theme, this one should be the standard dark theme. 

**Describe the solution you'd like**
I like dark themes but a gray theme is missing (please see the example  link at bottom page), bc nicely readable by day and night. I see the dark+blue theme as replacement for the default dark theme.  

**Describe alternatives you've considered**
The 3rd theme could be a user theme, eg. a css file, if you don't want to add a gray theme. 

**Additional context**
[Example theme - Gimp](https://raw.githubusercontent.com/Draekko-RAND/gimp-themes/gimp-2_10/screenshots/sog.png)

**Comment by permafrozen** [comment]
Hello, I am not sure if it helps, but you can style the program using your gtk theme. For example:
![satty-2024-11-10_00:07:59](https://github.com/user-attachments/assets/48d48b88-4e4e-4b3e-9dc2-4f11a6615346)

**Comment by Jopp-gh** [comment]
Nobody else interested in a middle-ground theme instead of radical white and black themes?

Would be very nice to modify themes through css
